docker/docker-compose.yml

```diff
@@ -114,6 +114,9 @@ services:
       - "traefik.port=8080"
       - "traefik.protocol=https"
       - "traefik.docker.network=proxy"
+      - "traefik.uploads.frontend.redirect.regex=^https?://(.*?).altis.dev/uploads/(.*)"
+      - "traefik.uploads.frontend.redirect.replacement=https://s3-${COMPOSE_PROJECT_NAME:-default}.altis.dev/uploads/$$2"
```

Comment:
Should this use $$1 instead?

Reply:
no, I made that error first time round - there are 2 capturing brackets

Reply:
Sorry, I mean instead of the environment variable, not instead of the path replacement

Reply:
No, it's intentional. The S3 container URL will always be that